src/app/code/community/Shopgate/Cloudapi/etc/config.xml

```diff
@@ -80,6 +89,9 @@
                 <shopgate_cloudapi_checkout>
                     <file>shopgate/cloudapi/checkout.xml</file>
                 </shopgate_cloudapi_checkout>
+                <shopgate_cloudapi_register>
```

Comment:
I think this should be inside the customer.xml file @andre-kraus created in his story as this is customer related, not register related.